src/web/lib/components/PaginatedThemeSelector/index.js

```diff
@@ -20,6 +28,14 @@ export const PaginatedThemeSelector = ({
   images = []
 }) => {
   const itemCount = themes.length;
+  const previousCount = usePrevious(themes.length);
+
+  React.useEffect(() => {
+  if (itemCount !== previousCount && previousCount !== null && itemCount % perPage === 0 && currentPage !== 0) {
```

Comment:
Indentation looks off.

Do you really need the `usePrevious` hook? Wouldn't the issue be fixed if you use
```
if (itemCount && currentPage >= itemCount) {
  setCurrentPage(itemCount - 1);
}
```

(I don't know if it is possible, but if there was a bulk-delete functionality, then the current proposal would fail again, whereas the proposed snippet would not break)